src/backtrace.c

```diff
@@ -509,7 +514,7 @@ static void backtrace_foreach(void (*cb)(void *arg, void *ptr), void *arg)
                         return;
                     }
                 }
-                // FALLTHROUGH!
+                __attribute__((fallthrough));
```

Comment:
I thought this was matched by GCC, but I guess if you changed it, it wasn't?

I think it should match this regex:

```
[ \t.!]*(ELSE,? |INTENTIONAL(LY)? )?
FALL(S | |-)?THR(OUGH|U)[ \t.!]*(-[^\n\r]*)?```

which is one of the various used by `-Wimplicit-fallthrough`